nni/compression/pytorch/compressor.py

```diff
@@ -589,8 +589,26 @@ class QuantGrad(torch.autograd.Function):
     """
     Base class for overriding backward function of quantization operation.
     """
+    @classmethod
+    def _quantize(cls, x, scale, zp):
```

Comment:
Suggest changing zp to zero_point to keep consensus.

Reply:
yeah, changed already